Automattic/jetpack
# Recommendations: improve usability of Recommendations' dismiss options

#### Changes proposed in this Pull Request:

This PR aims to make it easier for site owners to understand that they can dismiss the Recommendations banner appearing on their site after a successful connection to WordPress.com:
﻿- I've added a "dismiss" text next to the existing dismiss icon.
﻿- I've restyled that dismiss option based on the discussions below.
- ~~I've added a new "Not now" option to the main banner.~~

**Before**

![image](https://user-images.githubusercontent.com/426388/111286674-b9512b00-8642-11eb-9299-db0cdcc49da7.png)


**After**

![image](https://user-images.githubusercontent.com/426388/112370866-32552000-8cde-11eb-9363-cd7f7ce97510.png)



#### Jetpack product discussion

* p1614892057083100-slack-C0299DMPG

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Connect a brand new Jetpack site to WordPress.com.
* When coming back to your site post-connection, go to the Plugins page or the main dashboard page.
* Check banner and try to use one of the dismiss options
    * The banner should remain dismissed.
